corehq/ex-submodules/phonelog/tests/test_sumologic.py

```diff
@@ -76,7 +76,7 @@ def test_forceclose(self):
         at android.app.Activity.performCreate(Activity.java:6251)
         at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1107)
         at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2369)
-        ... 9 more] [device_model=Nexus 7] [android_version=6.0.1] [app_id=73d5f08b9d55fe48602906a89672c214] """
+        ... 9 more] [device_model=None] [android_version=None] [app_id=73d5f08b9d55fe48602906a89672c214] """
```

Comment:
I'm not familiar with sumologic. Can you say a bit about why you make this change? I thought after you correct the source of `device_model` and `android_version`, they should not be `None`.

Reply:
Good point. So, when it comes to mobile, we need to cater not only for the version with a particular feature/change but also for previous versions as we don't know when all the users will be on the latest version. For instance, this change will be shipped with CommCare 2.57 which is to be deployed at the end of Q2, this means that until then, all the users' device logs won't include these new fields. And once released, we will continue having users in the previous version for some time.